### PR TITLE
Dispatch cmmn engine events when task is assigned #2744

### PR DESCRIPTION
**Pull request by DavidPLamas**
#### Check List:
* Unit tests: YES
* Documentation: NO

### CODE REVIEW

File: modules/flowable-cmmn-api/src/main/java/org/flowable/cmmn/api/event/FlowableTaskAssignedEvent.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package org.flowable.cmmn.api.event;
```
Comment (dbmalkovsky) [inline modules/flowable-cmmn-api/src/main/java/org/flowable/cmmn/api/event/FlowableTaskAssignedEvent.java:1]: Please add the standard license header to all new files; thanks.

**Review by filiphr — CHANGES_REQUESTED** [review]
Thanks for the PR @DavidPLamas. 

It looks great, I have only one small remark. I don't think that we need a new `FlowableTaskAssignedEvent` class since there are a lot of other places where we throw the task assigned event with a `FlowableEngineEntityEvent` instead. 

Can you please adapt your changes to match the way it is done currently?

**Comment by DavidPLamas** [comment]
Hi @filiphr,

I've removed the FlowableTaskAssignedEvent and started using FlowableEntityEvent instead. I was gonna copy the code from ```org.flowable.engine.delegate.event.impl.FlowableEventBuilder#createEntityEvent``` but I thought it was too much.
Let me know if there is anything else that needs changing.

**Comment by DavidPLamas** [comment]
Hi,
It's been a while, I just noticed the build has failed. From looking at the error log, it doesn't seem to be because of my changes.
Is there any way to retrigger the build to check if it was just a temporary problem?

**Comment by filiphr** [comment]
Thanks for the ping @DavidPLamas. No need to retrigger. The Windows build is a bit unstable. The other builds look good

**Comment by filiphr** [comment]
I manually merged this in https://github.com/flowable/flowable-engine/commit/86f818f37eca2f2c06ea2e66f1340ba51e0308db

**Pull request closed without merging**